src/update_lmod_system_cache_files.in

```diff
@@ -334,7 +334,7 @@ set_defaults
 check_parameters
 
 if [ -z "$LMOD_REVERSEMAPT_DIR" ]; then
-  LMOD_REVERSEMAPT_DIR="$LMOD_CACHE_DIR/../reverseMapD"
+  LMOD_REVERSEMAPT_DIR="$LMOD_CACHE_DIR/reverseMapD"
```

Comment:
Also, I don't understand what the logic is behind this choice? We have a `$LMOD_CACHE_DIR`, why not put everything below it?